src/pcapi/core/bookings/api.py

```diff
@@ -104,35 +102,40 @@ def book_offer(
             if FeatureToggle.AUTO_ACTIVATE_DIGITAL_BOOKINGS.is_active():
                 booking.mark_as_used()
 
+        individual_booking = IndividualBooking(userId=beneficiary.id, booking=booking)
         stock.dnBookedQuantity += booking.quantity
 
-        repository.save(booking, stock)
+        repository.save(individual_booking, stock)
```

Comment:
Je suppose que SQLAlchemy s'en sort et insère `booking` de façon transparente, mais je l'ajouterais bien explicitement ici, quand même, pour clarifier les objects insérés ou mis à jour :

```suggestion
        repository.save(booking, individual_booking, stock)
```

Reply:
@dbaty, Je vais merger comme tel, suite à échange avec Cyril. En effet, le but est d'aller vers l'utilisation exclusive de `IndividualBooking` et `EducationalBooking`, `booking` ne devenant plus qu'un "simple" attribut des deux classes métiers.
Pas de souci pour échanger davantage sur le sujet; toute façon avec l'incident CIrcleCi, je vais attendre avant de merger; sinon n'hésite pas à nous pinguer sur slack!